datasets/mslr2022/mslr2022.py

```diff
@@ -0,0 +1,176 @@
+this task, and to encourage the development of modeling contributions, scaffolding tasks, methods
+for model interpretability, and improved automated evaluation methods in this domain.
+"""
+
+
+import os
+
+import pandas as pd
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{DeYoung2021MS2MS,
+    title        = {MSˆ2: Multi-Document Summarization of Medical Studies},
+    author       = {Jay DeYoung and Iz Beltagy and Madeleine van Zuylen and Bailey Kuehl and Lucy Lu Wang},
+    booktitle    = {EMNLP},
+    year         = {2021}
+}
+@article{Wallace2020GeneratingN,
+    title        = {Generating (Factual?) Narrative Summaries of RCTs: Experiments with Neural Multi-Document Summarization},
+    author       = {Byron C. Wallace and Sayantani Saha and Frank Soboczenski and Iain James Marshall},
+    year         = 2020,
+    journal      = {AMIA Annual Symposium},
+    volume       = {abs/2008.11293}
+}
+"""
+
+_DATASETNAME = "mslr2022"
+
+_DESCRIPTION = """\
+The Multidocument Summarization for Literature Review (MSLR) Shared Task aims to study how medical
+evidence from different clinical studies are summarized in literature reviews. Reviews provide the
+highest quality of evidence for clinical care, but are expensive to produce manually.
+(Semi-)automation via NLP may facilitate faster evidence synthesis without sacrificing rigor.
+The MSLR shared task uses two datasets to assess the current state of multidocument summarization
+for this task, and to encourage the development of modeling contributions, scaffolding tasks, methods
+for model interpretability, and improved automated evaluation methods in this domain.
+"""
+
+_HOMEPAGE = "https://github.com/allenai/mslr-shared-task"
+
+_LICENSE = "Apache-2.0"
+
+_URLS = {
+    _DATASETNAME: "https://ai2-s2-mslr.s3.us-west-2.amazonaws.com/mslr_data.tar.gz",
+}
+
+
+class MSLR2022(datasets.GeneratorBasedBuilder):
+    """MSLR2022 Shared Task."""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="ms2",
+            version=VERSION,
+            description="This dataset consists of around 20K reviews and 470K studies collected from PubMed. For details on dataset contents and construction, please read the MS^2 paper (https://arxiv.org/abs/2104.06486).",
+        ),
+        datasets.BuilderConfig(
+            name="cochrane",
+            version=VERSION,
+            description="This is a dataset of 4.5K reviews collected from Cochrane systematic reviews. For details on dataset contents and construction, please read the AMIA paper (https://arxiv.org/abs/2008.11293).",
+        ),
+    ]
+
+    def _info(self):
+        fields = {
+            "review_id": datasets.Value("string"),
+            "pmid": datasets.Sequence(datasets.Value("string")),
+            "title": datasets.Sequence(datasets.Value("string")),
+            "abstract": datasets.Sequence(datasets.Value("string")),
```

Comment:
Maybe rename this "titles" and "abstracts" since there are multiple documents ?

Reply:
I was following the convention of other summarization datasets I see on the hub, e.g. [multi-news](https://huggingface.co/datasets/multi_news) and [multi-x-science-sum](https://huggingface.co/datasets/multi_x_science_sum) but happy to change if you like!